saleor/graphql/order/mutations/orders.py

```diff
@@ -677,16 +678,14 @@ def get_instance(cls, info, **data):
         return instance
 
     @classmethod
-    @traced_atomic_transaction()
-    def perform_mutation(cls, root, info, **data):
-        order = cls.get_instance(info, **data)
-        order.status = OrderStatus.UNFULFILLED
-        order.save(update_fields=["status"])
-        payment = order.get_last_payment()
-        manager = info.context.plugins
+    def process_payment(cls, order, payment, info, manager, to_pay):
         if payment and payment.is_authorized and payment.can_capture():
```

Comment:
You can also drop `payment.is_authorized` as it is covered by `can_capture`

Reply:
OK